packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -298,6 +306,258 @@ describe('LedgerKeyAgent', () => {
+            )
+            .build();
+
+          await expect(tx.sign()).rejects.toThrow(
+            InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_STAKE_CREDENTIAL_ONLY_AS_PATH
+          );
+        });
+
+        it('can sign a transaction with VoteDelegation certs', async () => {
+          const credential = getStakeCredential((await firstValueFrom(wallet.delegation.rewardAccounts$))?.[0].address);
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.VoteDelegation,
+              dRep: credential,
+              stakeCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          // expect(await tx.sign()).toBeTruthy();
+          // TODO: should be able to sign?
+          await expect(tx.sign()).rejects.toThrow(
+            'HwMappingError: Hardware data mapping failure: Ledger computed a different transaction id'
+          );
+        });
+
+        it('would throw while trying to sign a transaction with VoteDelegation certs', async () => {
+          const credential = getStakeCredential(
+            Cardano.RewardAccount('stake_test1up7pvfq8zn4quy45r2g572290p9vf99mr9tn7r9xrgy2l2qdsf58d')
+          );
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.VoteDelegation,
+              dRep: credential,
+              stakeCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          await expect(tx.sign()).rejects.toThrow(
+            InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_STAKE_CREDENTIAL_ONLY_AS_PATH
+          );
+        });
+
+        it('can sign a transaction with AuthorizeCommitteeHot certs', async () => {
+          const credential = getStakeCredential((await firstValueFrom(wallet.delegation.rewardAccounts$))?.[0].address);
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.AuthorizeCommitteeHot,
+              coldCredential: credential,
+              hotCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          // expect(await tx.sign()).toBeTruthy();
+          // TODO: should be able to sign?
```

Comment:
@mchappell @AngelCastilloB exact error says `Action rejected by Ledger's security policy`  (mapped out [here](https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/blob/master/src/errors/invalidDataReason.ts#L247C20-L247C63)), not sure if it's me or the `AuthorizeCommitteeHot` is not supported in general

Reply:
Do you have the latest version of the ledger firmware?, its probably best con ask the team behind the Ledger HW app